### PR TITLE
Adds JSDoc

### PR DESCRIPTION
**Pull request by robertt**
I'm bored on my train and on my phone so I don't know what to do. Please  don't decline because you don't like me Josef.

### CODE REVIEW

**Review by jb3 — CHANGES_REQUESTED** [review]
Comments left

File: lib/webhook.js [hunk 1]
```diff
@@ -32,13 +38,20 @@ class Webhook {
...
 
+    /**
+    @param {string} name The username of the error webhook.
+    @param {string} message The message of the errorwebhook.
+    */
     err(name, message) {
-        let payload = {
-            "username": name,
+        let payload= {
+            "username": names,
```
[the remark this answers is not in the document]
Comment (Haxified1) [reply lib/webhook.js]: oops

[the remark this answers is not in the document]
Comment (Haxified1) [reply lib/webhook.js]: typo

**Comment by Haxified1** [comment]
Look good? My phone typing is not great :/

File: lib/webhook.js [hunk 2]
```diff
@@ -32,12 +38,19 @@ class Webhook {
...
         return `${endpoint}/${this.id}/${this.token}/slack`
     }
 
+    /**
+    @param {string} name The username of the error webhook.
+    @param {string} message The message of the errorwebhook.
+    */
     err(name, message) {
-        let payload = {
+        let payload= {
```
Comment (JoeBanks13) [inline lib/webhook.js]: Space here should not have been removed

**Review by jb3 — CHANGES_REQUESTED** [review]
Just need the following addressing, rest of it looks good

**Comment by Haxified1** [comment]
Done

**Comment by Haxified1** [comment]
Fixed another smol thing.

**Review by jb3 — APPROVED** [review]

**Comment by JoeBanks13** [comment]
Looks good to me, cheers for that one :+1:

File: lib/webhook.js (hunk 2 above)
Comment (Mstone87) [reply lib/webhook.js]: Sugabitcoin

**Pull request merged**